generators/input-plugin/templates/_package.json

```diff
@@ -21,12 +22,13 @@
   "license": "Apache-2.0",
   "dependencies": {},
   "devDependencies": {
-    "ava": "^0.12.0",
-    "eslint": "2.2.0",
-    "eslint-config-punchcard": "^0.1.2",
+    "ava": "^0.14.0",
+    "coveralls": "^2.11.9",
+    "eslint": "^2.10.0",
+    "eslint-config-punchcard": "^1.0.1",
     "nyc": "^6.0.0",
     "punchcard-content-types": "^0.1.0",
-    "tap-diff": "^0.1.1"
+    "tap-diff": "^0.1.1",
```

Comment:
@rachelnicole please remove this comma. that trailing comma linter does not apply to `package.json` and this trailing comma will most likely cause errors.